### DISCUSSION TITLE
Install external package

### DISCUSSION THREAD

**Discussion by TrentinErasmo** [iraf-community/iraf#142] [Q&A] [answered]
I have installed Iraf using this guide : https://iraf-community.github.io/install.html
I need to install an external package (sptable) but when I go the "extern" directory and try the command: "$ make self_update" or "$ make sptable" I get this message error:
"make: *** No rule to make target 'sptable (or self_update)'. Stop"
How can I do?
Thanks in advance

**Comment by olebole** [comment]
We don't have a repository of precompiled external packages here, so you need to compile the package yourself and put it into the `extern` directory. The "sptable" package is in the [iraf-community/sptable](https://github.com/iraf-community/iraf-sptable) repository.
If you use Debian or a derivative (Ubuntu, Mint etc.), you can install a selection of external packages with apt.

**Comment by TrentinErasmo** [comment]
I am sorry to disturb again, I am pretty new with Iraf. I had already downloaded the source file of sptable but I need a guide in order to compile the package.
I am using Ubuntu 19.10

**Comment by olebole** [comment, marked as answer]
On Ubuntu, you can just do
```
apt install iraf iraf-sptable
```
to install IRAF and sptable. `irafcl` is the command to start the IRAF shell there, or you also install `python3-pyraf` and use the `pyraf` command.
If you are new to IRAF, I would however recommend to consider using a modern alternative like Astropy (with its affiliated packages) instead.

**Comment by TrentinErasmo** [comment]
Thank you very much!
I am also checking for Astropy and pyraf, thanks for the advice.
